apps/dashboard/src/components/side-navigation/organization-dropdown.tsx

```diff
@@ -32,7 +32,7 @@ export const OrganizationDropdown = () => {
     <ClerkOrganizationSwitcher
       hidePersonal
       skipInvitationScreen
-      afterCreateOrganizationUrl={ROUTES.USECASE_SELECT + '?v2_opt_in=true'}
+      afterCreateOrganizationUrl={ROUTES.USECASE_SELECT}
```

Comment:
the opt-in logic is now handled with the Clerk webhook event: https://github.com/novuhq/packages-enterprise/pull/270

Reply:
Could it be that there will be some delivery delay on their webhook? It happened before, and during that time our service might redirect the user to the wrong place. How can we ensure that by the time the user reaches the USECASE SELECT he already opted in?

Reply:
@scopsy The user is `opted in` after the questionnaire form is submitted, which is shown after the org is created. Why would we need to do this once again after the org is created from the orgs switcher/select?